packages/ios_platform_images/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.2.0+7
+
+* Ignore deprecation of `load` method in [ImageProvider] so it can be
+  deprecated at the framework level, without breaking CI.
```

Comment:
Since this is primarily client-facing, I would just say "Ignores the warning for the upcoming deprecation of `ImageProvider.load`."

Reply:
Much better, replaced by your version!